src/components/filter/Filter.tsx

```diff
@@ -29,21 +30,80 @@ type Props = {
 export const Filter = ({
   filterTypes,
   initialFilters,
+  initialStartDate,
   isOverlay = false,
   queryVariables,
   setQueryVariables,
   withSearch = false
 }: Props) => {
-  const [filters, setFilters] = useState<FilterProps>(queryVariables);
+  const [filters, setFilters] = useState<FilterProps>(
+    initialStartDate
+      ? Object.fromEntries(
+          Object.entries(queryVariables).filter(
+            ([key, value]) => !(key === 'start_date' && value === initialStartDate)
+          )
+        )
+      : queryVariables
+  );
   const [isCollapsed, setIsCollapsed] = useState(true);
   const [filterCount, setFilterCount] = useState(0);
 
   useEffect(() => {
     if (!isOverlay) {
-      setQueryVariables((prev) => ({ search: prev.search || '', ...filters }));
+      setQueryVariables((prev) => {
+        const newFilters = { ...filters };
+
+        if (newFilters.start_date === initialStartDate) {
+          delete newFilters.start_date;
+          return {
+            ...prev,
+            search: prev.search || '',
+            start_date: initialStartDate,
+            ...newFilters
+          };
+        }
+
+        if (!newFilters.start_date) {
+          return {
+            ...prev,
+            search: prev.search || '',
+            start_date: initialStartDate,
+            ...newFilters
+          };
+        }
+
+        return { ...prev, search: prev.search || '', ...newFilters };
+      });
     }
   }, [filters]);
 
+  const resetFilters = () => {
+    if (!isOverlay) {
+      setIsCollapsed(!isCollapsed);
+
+      setTimeout(() => {
+        setFilters(
+          initialStartDate
+            ? Object.fromEntries(
+                Object.entries(queryVariables).filter(
+                  ([key, value]) => !(key === 'start_date' && value === initialStartDate)
+                )
+              )
+            : queryVariables
+        );
+
+        setQueryVariables({
+          ...queryVariables,
```

Comment:
if this is a rest, shouldn't it also reset the query variables and not use the latest ones?

Reply:
I don't really understand the question.

Reply:
you are setting query variables with the current values of `queryVariables`. this can be edited already, so this has not initial values anymore and seems not to be a reset?

Reply:
I think there is a naming error here. This is actually an `initialQueryVariable`. So it's right to reset.

```
queryVariables={initialQueryVariables}
```

Reply:
where is this line from?

Reply:
SueListScreen -> Filter component (Line: 323)

Reply:
shouldn't it be `initialFilters` in that case?

Reply:
You are absolutely right. But I think there's a reason why we don't have this nomenclature.

Reply:
shouldn't it be the `initialQueryVariables` here to reset to, instead of the `queryVariables`

Reply:
That's the only confusion left. No problem for the overlay filter. But for the SUE filter the `queryVariables` are actually `initialQueryVariables`.